packages/components/src/filters/advanced/date-filter.js

```diff
@@ -186,16 +186,20 @@ class DateFilter extends Component {
 	}
 
 	render() {
-		const { config, filter, isEnglish } = this.props;
+		const { className, config, filter, isEnglish } = this.props;
 		const { rule } = filter;
 		const { labels, rules } = config;
 		const screenReaderText = this.getScreenReaderText( filter, config );
 		const children = interpolateComponents( {
-			mixedString: labels.title,
+			mixedString: sprintf(
+				'{{title}}%s{{/title}} {{rule /}} {{filter /}}',
```

Comment:
Missed this before - these `mixedStrings` were being passed through the translation function `__()` before.. unless we don't actually need that for RTL support, I think that needs to get restored.

Reply:
You're totally right. Moved the placeholders back to the respective config.js files and included the new `title` placeholder there.